Target.java

```diff
@@ -1,17 +1,59 @@
-/*
- * File: Target.java
- * Name: 
- * Section Leader: 
- * -----------------
- * This file is the starter file for the Target problem.
- */
-
-import acm.graphics.*;
-import acm.program.*;
-import java.awt.*;
-
-public class Target extends GraphicsProgram {	
-	public void run() {
-		/* You fill this in. */
-	}
-}
+/*
+ * File: Target.java
+ * Name: 
+ * Section Leader: 
+ * -----------------

```

Comment:
This is a great attempt because this solution is scalable. If the user decide to modify the radius of the outer circle for example, you wouldn't need to in your code and try to modify it. Instead it will just do the right thing.

However, your solution doesn't position the target at the center. You should have specified where the circles should be drawn in your "add" method.